toolchain/language_server/testdata/text_document/open_with_cpp_nonexistent.carbon

```diff
@@ -27,17 +27,6 @@
 
 // --- AUTOUPDATE-SPLIT
 
-// CHECK:STDERR: /test.carbon: warning: dropping diagnostic in /test.carbon.generated.cpp_imports.h:
```

Comment:
Thanks! Can you also remove the TODO on lines 5-8?

Reply:
Done.